diff --git a/services/camera/libcameraservice/api1/client2/Parameters.cpp b/services/camera/libcameraservice/api1/client2/Parameters.cpp
index 09db25f..da92d65 100644
--- a/services/camera/libcameraservice/api1/client2/Parameters.cpp
+++ b/services/camera/libcameraservice/api1/client2/Parameters.cpp
@@ -2961,10 +2961,6 @@ status_t Parameters::getFilteredSizes(Size limit, Vector<Size> *sizes) {
                 sc.width <= limit.width && sc.height <= limit.height) {
             int64_t minFrameDuration = getMinFrameDurationNs(
                     {sc.width, sc.height}, HAL_PIXEL_FORMAT_IMPLEMENTATION_DEFINED);
-            if (minFrameDuration > MAX_PREVIEW_RECORD_DURATION_NS) {
-                // Filter slow sizes from preview/record
-                continue;
-            }
             sizes->push({sc.width, sc.height});
         }
     }
